tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -1151,6 +1151,66 @@ fillSymbolInfo(CursorSymbolInfo &Symbol, const DeclInfo &DInfo,
   return llvm::Error::success();
 }
 
+// If E is a literal, returns the declaration that should be reported by
```

Comment:
Nitpick, you can mark `E` as a parameter.

```suggestion
// If \p E is a literal, returns the declaration that should be reported by
```